src/server/api.php

```diff
@@ -1005,18 +1004,14 @@ function get_covidcast_meta_cached() {
   $epidata = null;
   $result = mysqli_query($dbh, $query);
   if($row = mysqli_fetch_array($result)) {
+    // parse and use the cached response
+    $epidata = json_decode($row['epidata'], true);
+
     if (intval($row['age']) < $max_age && strlen($row['epidata']) > 0) {
```

Comment:
`$max_age` is no longer defined, since line 998 was deleted above. Also, this seems to issue an error if the row is too *new*, not too *old*